src/main.cpp

```diff
@@ -87,6 +87,7 @@ void loop()
     spi_init();
     xmodem_receive();
     // Fallthrough
```

Comment:
could you remove this comment? we are not falling through anymore....i know its a nit-pick but its bothering me :P